packages/es-ingest/src/main/java/software/uncharted/terarium/esingest/ElasticIngestApplication.java

```diff
@@ -0,0 +1,67 @@
+package software.uncharted.terarium.esingest;
+
+import java.lang.reflect.Constructor;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.PropertySource;
+
+import lombok.extern.slf4j.Slf4j;
+import software.uncharted.terarium.esingest.configuration.Config;
+import software.uncharted.terarium.esingest.ingests.IElasticIngest;
+import software.uncharted.terarium.esingest.service.ElasticIngestParams;
+import software.uncharted.terarium.esingest.service.ElasticIngestService;
+
+@SpringBootApplication
+@Slf4j
+@PropertySource("classpath:application.properties")
+public class ElasticIngestApplication {
+
+	@Autowired
```

Comment:
Do we not have a .editorconfig that sets the indent to something sane like 2 (or maybe 4)?  8 feels like a timewarp back to the 1800s of ludicrous indents, next we will be wearing knicker-bockers

Reply:
It should be whatever the tab indent is set to. I think github just shows them as 8. Its 2 in my editor.